doc/reference/configuration/configuration_reference.rst

```diff
@@ -221,6 +488,718 @@ This section describes options related to :ref:`storing configuration in etcd <c
+        This option is added to keep the compatibility with the current versions of Cartridge and might be removed in the future.
+
+    | Type: string
+    | Default: ``auto``
+    | Environment variable: TT_REPLICATION_BOOTSTRAP_STRATEGY
+
+
+.. _configuration_reference_replication_connect_timeout:
+
+.. confval:: replication.connect_timeout
+
+    A timeout (in seconds) a replica waits when trying to connect to a master in a cluster.
+    See :ref:`orphan status <replication-orphan_status>` for details.
+
+    This parameter is different from
+    :ref:`replication.timeout <configuration_reference_replication_timeout>`,
+    which a master uses to disconnect a replica when the master
+    receives no acknowledgments of heartbeat messages.
+
+    | Type: number
+    | Default: 30
+    | Environment variable: TT_REPLICATION_CONNECT_TIMEOUT
+
+
+.. _configuration_reference_replication_election_mode:
+
+.. confval:: replication.election_mode
+
+    A role of a replica set node in the :ref:`leader election process <repl_leader_elect>`.
+
+    The possible values are:
+
+    *   ``off``
+
+        A node doesn't participate in the election activities.
+
+    *   ``voter``
+
+        A node can participate in the election process but can't be a leader.
+
+    *   ``candidate``
+
+        A node should be able to become a leader.
+
+    *   ``manual``
+
+        Allow to control which instance is the leader explicitly instead of relying on automated leader election.
+
+        By default, the instance acts like a voter -- it is read-only and may vote for other candidate instances.
+        Once :ref:`box.ctl.promote() <box_ctl-promote>` is called, the instance becomes a candidate and starts a new election round.
+        If the instance wins the elections, it becomes a leader but won't participate in any new elections.
+
+    | Type: string
+    | Default: box.NULL
+    | Environment variable: TT_REPLICATION_ELECTION_MODE
+
+
+.. _configuration_reference_replication_election_timeout:
+
+.. confval:: replication.election_timeout
+
+    Specifies the timeout (in seconds) between election rounds in the
+    :ref:`leader election process <repl_leader_elect>` if the previous round
+    ended up with a split vote.
+
+    It is quite big, and for most of the cases, it can be lowered to
+    300-400 ms.
+
+    To avoid the split vote repeat, the timeout is randomized on each node
+    during every new election, from 100% to 110% of the original timeout value.
+    For example, if the timeout is 300 ms and there are 3 nodes started
+    the election simultaneously in the same term,
+    they can set their election timeouts to 300, 310, and 320 respectively,
+    or to 305, 302, and 324, and so on. In that way, the votes will never be split
+    because the election on different nodes won't be restarted simultaneously.
+
+    | Type: number
+    | Default: 5
+    | Environment variable: TT_REPLICATION_ELECTION_TIMEOUT
```

Comment:
Don't know why, but these blocks aren't splitted on the website.

![image](https://github.com/tarantool/doc/assets/92749/2147ee1b-9fdd-4024-8505-4f64acc53b33)

The same for the next option.

Reply:
Looks like a vertical pipe (`|`) without any content add an empty string between blocks.